web/client/jsapi/MapStore2.js

```diff
@@ -93,7 +94,7 @@ let stateChangeListeners = [];
 const getInitialActions = (options) => {
     if (!options.initialState || !options.initialState.defaultState.map) {
         if (options.configUrl) {
-            return [loadMapConfig.bind(null, options.configUrl || defaultConfig, options.mapId)];
+            return [initMap.bind(null), loadMapConfig.bind(null, options.configUrl || defaultConfig, options.mapId)];
```

Comment:
could it be `initMap` instead of `initMap.bind(null)`

Reply:
```suggestion
            return [initMap, loadMapConfig.bind(null, options.configUrl || defaultConfig, options.mapId)];
```